terraform/main.go

```diff
@@ -22,56 +26,103 @@ var tf_envs_names []string = []string{
 	filepath.Join(tf_cwd_dir, "creds.tfvars.json"),
 }
 
-func main() {
-	logging.InitLogger()
-	log.Info().Msg("Starting GoTerra")
-
-	//	ihpTfBin := ihp.NewTfBin(tf_bin_dir, tf_version, tf_cwd_dir, tf_envs_names)
-
-	mq := mqhandler.MqHandler{}
-	mq.Init()
+var ihp_presets = map[string]int{
+	"mysql": 1,
+}
 
+func listenForEvents(mq *mqhandler.MqHandler, mq_preset_container_events *[]mqhandler.PresetContainerStartEvent) {
 	for {
 		select {
 		case event := <-mq.CustomContainerStartEventChannel:
 			// TODO: do something with this
 			log.Debug().Msgf("Received custom start event: %v", event)
+			log.Warn().Msg("CustomContainerStartEventChannel not implemented")
 		case event := <-mq.PresetContainerStartEventChannel:
-			// TODO: do something with this
-			log.Debug().Msgf("Received preset start event: %v", event)
-		case event := <-mq.StopContainerEventChannel:
+			event_sanitized := event
+			event_sanitized.ContainerEnv = map[string]string{"<redacted>": "<redacted>"} // Redact sensitive data
+			log.Debug().Msgf("Received preset start event: %v", event_sanitized)
+			*mq_preset_container_events = append(*mq_preset_container_events, event)
+		case event := <-mq.DestroyContainerEventChannel:
 			// TODO: do something with this
 			log.Debug().Msgf("Received delete event: %v", event)
+			log.Warn().Msg("CustomContainerStartEventChannel not implemented")
 		}
 	}
+}
 
-	/*
+func main() {
+	logging.InitLogger()
+	log.Info().Msg("Starting GoTerra")
 
-		uidsToRemove := []int{}
+	ihpTfBin := ihp.NewTfBin(tf_bin_dir, tf_version, tf_cwd_dir, tf_envs_names)
+	ihpTfBin.InitTerraform()
+
+	// Set env var
+	os.Setenv("MQ_CONNECTION_STRING", "amqp://guest:guest@localhost:5672/")
```

Comment:
At least add a check before setting it. This will overwrite any var set within the env/the container down the road